tools/gce-google-keys-to-cmek/main.py

```diff
@@ -101,28 +109,32 @@ def migrate_instance_to_cmek(project, zone, instance, key_ring, key_name,
   for source_disk in disks:
     disk_regexp = r'^https:\/\/www\.googleapis\.com\/compute\/v1\/projects\/(.*?)\/zones\/(.*?)\/disks\/(.*?)$'
```

Comment:
I would move the pattern to a module-level constant (and maybe use split instead of a regexp anyway)